### DISCUSSION TITLE
A few feature suggestions for Hippoplayer!

### DISCUSSION THREAD

**Discussion by joonas-foo** [emoon/HippoPlayer#213] [Ideas]
Here are a few ideas and suggestions for Hippoplayer! I wrote this earlier to Strawberry's forum and later to Clementine's Github discussions page. Big thanks for updating the player, I haven't got to test it yet as I ran into some problems trying to install it. I'm a novice Linux and Ubuntu user! :-)

-Support for Spotify, Deezer, Qobuz, Tidal, Youtube, Vimeo, Soundcloud, Mixcloud, Hearthis.at, Bandcamp, Apple music. A simple (meta)search bar for these, you would need to have an account on the sites that require it, maybe a simple browser window without needless graphics and other bloat if necessary. Also banner or text ads would be compulsory for the commercial streaming sites.

-Support for Discogs, Allmusic, Wikipedia for information about the currently playing song/album.

-Deadhead plugin, plays Grateful dead's and a few other band's (Furthur, Mickey Hart band, Phil lesh and friends, Ratdog) music from archive.org.
https://web.archive.org/web/20201017234651/https://sites.google.com/site/arodsisland/deadhead-help
https://www.facebook.com/profile.php?id=100063552248155

-Winamp's pitchfork like plugin would be excellent,
https://web.archive.org/web/20221007101002/http://www.mpx.net/software/details/pitchfork.html
https://jevon.org/wiki/Pitchfork_for_Winamp

-An alarmclock function, to wake up to music!

-A search bar for http://radio.net/, it lists some 60000 internet radiostations. The site has went to much worse lately tho.

-A bpm counter, possibly as an option.

-Last.fm and Musicbrainz support, possibly Libre.fm, Maloja, Webhook and Pleroma support! I only use the first two, I don't know much about the latter.

**Comment by emoon** [comment]
Hi,

Thanks for the suggestion, and while some of these may fall into the space of the player, most of them don't. As stated in the readme:
> HippoPlayer is a music player for macOS, Windows and Linux that mostly focuses on music made for systems such as Amiga, C64, and gaming systems. While HippoPlayer supports MP3, FLAC, Ogg Vorbis, etc it's not the main target.
